CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ etcd container with this new field.
 ### Changed
 
 - Example deployments pin to the released version of the operator image
+- Service account of operator pod will be passed to backup pod.
+- Downward API of namespace and name is a requirement to start etcd operator pod.
```

Comment:
Downward API of namespace and pod name is required to start the etcd operator pod. `[example] [link to example]`